### PR TITLE
sync to the latest github/linguist v7.9.0

### PR DESCRIPTION
**Pull request by lafriks**

### CODE REVIEW

File: internal/code-generator/generator/generator.go [hunk 1]
```diff
@@ -32,12 +34,17 @@ func executeTemplate(w io.Writer, name, path, commit string, fmap template.FuncM
 	getCommit := func() string {
 		return commit
 	}
+	stringVal := func(val string) string {
```
Comment (bzz) [inline internal/code-generator/generator/generator.go]: As this seems to be the definition of this function used in templates, it would be nice to document the purpose of it here.

File: README.md [hunk 2]
```diff
@@ -118,6 +118,8 @@ Parsing [linguist/samples](https://github.com/github/linguist/tree/master/sample
 
 * [Heuristics for ".rno" extension](https://github.com/github/linguist/blob/3a1bd3c3d3e741a8aaec4704f782e06f5cd2a00d/lib/linguist/heuristics.yml#L365) in RUNOFF could not be parsed, due to unsupported lookahead in RE2 regexp engine.
 
+* [Heuristics for ".ino" extension](https://github.com/github/linguist/blob/f0e2d0d7f1ce600b2a5acccaef6b149c87d8b99c/lib/linguist/heuristics.yml#L222) in NASL could not be parsed, due to unsupported possessive quantifier in RE2 regexp engine.
+
```
Comment (bzz) [inline README.md]: Nice catch! Wonder why this does not require adding any [corner case to the tests](https://github.com/go-enry/go-enry/blob/1ab8148c10379417036bc9d403386a8291ac63dc/common_test.go#L431) 🤔

**Comment by bzz** [comment]
Great job, thank you! Few minor comments above.

It would be a good time to 🔪 a next release as soon as I fix the CI and this and default color changes are merged.

File: README.md (hunk 2 above)
Comment (lafriks) [reply README.md]: all test cases are automatically used from linguist and does not need to be synced somehow?

File: internal/code-generator/generator/generator.go (hunk 1 above)
Comment (lafriks) [reply internal/code-generator/generator/generator.go]: done

File: README.md (hunk 2 above)
Comment (bzz) [reply README.md]: > all test cases are automatically used from linguist

that particular test mimics one the Linguist testing strategies by verifying the results of language prediction over all original Linguist files under `./samples`. Some of the incompatible regexps e.g in the heuristics matcher make some samples fail from time to time. 

I checked, and in this case https://github.com/github/linguist/pull/4792 there is no mention of `.ino` extension at all - I guess it's a typo in README and should be `.inc` instead?

I guess the answer to original question would be that the test
```
$ ENRY_TEST_REPO="${PWD}/.linguist/" go test . -run Test_EnryTestSuite -testify.m TestLinguistCorpus
```
pass, as all those 2 `.inc` examples of the NASL are not disambiguated by the heuristic strategy `GetLanguagesByContent` (that uses those regexps), but rather are ranked first by the very last BayesianClassifier instead.

Thus I would append `.inc` to the list of other similar cases in [this comment](https://github.com/go-enry/go-enry/blob/1ab8148c10379417036bc9d403386a8291ac63dc/common_test.go#L433) to simplify the maintenance once we have a failing tests :)

Comment (lafriks) [reply README.md]: yes, it is typo, should be `.inc`, will do

**Comment by bzz** [comment]
Let's rebase on latest master to get the CI green before merging it.

**Comment by lafriks** [comment]
Am I missing something? I rebased, rerun `make code-generate` but somehow I get different results :)

**Comment by lafriks** [comment]
ok, I don't know what helped but something to do with setting:
```
git config core.autocrlf false
git config core.eol lf
```
in `.linguist` directory and removing files and doing git reset --hard there, don't know :)

**Comment by bzz** [comment]
Which platform did you run code generation on?

**Comment by lafriks** [comment]
I use Linux
